checklist-pl.md

```diff
@@ -0,0 +1,27 @@
+## 1. Dozwolona składnia markdown na Devchecklists:
+  * [ ] Github usuwa białe znaki podczas kompilacji plików markdown. Aby utworzyć poprawny plik skorzystaj z [nieprzetworzonej wersji](https://raw.githubusercontent.com/vintasoftware/devchecklists-template/master/checklist-pl.md) tego pliku.
```

Comment:
The link does not work for english version. The Raw Button hiperlink is shorter

Compare with:
https://github.com/vintasoftware/devchecklists-template/blob/master/checklist-en.md

Reply:
I think that the link should start working after merge. I specified link to this repository but file doesn't exist before merge.

Reply:
Ok, got it :)